#ifndef DIALOG_VULNDETAILINFO_H
#define DIALOG_VULNDETAILINFO_H

#include <QDialog>
#include "shared_define.h"
namespace Ui {
class Dialog_VulnDetailInfo;
}

class Dialog_VulnDetailInfo : public QDialog
{
    Q_OBJECT

public:
    explicit Dialog_VulnDetailInfo(QWidget *parent = nullptr);
    ~Dialog_VulnDetailInfo();

    void set_vulnInfoStruct(VulnDetailInfo vulnDetailInfo);
private:
    Ui::Dialog_VulnDetailInfo *ui;
};

#endif // DIALOG_VULNDETAILINFO_H
